### PR TITLE
Hygiene fix in MQTT demos to log warning for unexpected PINGRESP packet event

### PR DESCRIPTION
**Pull request by aggarw13**
MQTT demos that use `MQTT_ProcessLoop` should not expect the notification of an incoming PINGRESP packet.

### CODE REVIEW

**Review by sarenameas — DISMISSED** [review]

File: FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/mqtt_demo_mutual_auth_config.h [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://www.FreeRTOS.org
+ * http://aws.amazon.com/freertos
+ *
+ */
+
+#ifndef DEMO_CONFIG_H
+#define DEMO_CONFIG_H
```
Comment (yourslab) [inline FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/mqtt_demo_mutual_auth_config.h:28]: This untracked file seems to have been pushed by mistake. Happens to me all the time too.

Comment (aggarw13) [reply FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/mqtt_demo_mutual_auth_config.h:28]: Oops, thanks for catching.

**Review by sarenameas — APPROVED** [review]

**Review by dan4thewin — APPROVED** [review]

**Review by yourslab — APPROVED** [review]

**Pull request merged**